NitroxServer/Server.cs

```diff
@@ -34,6 +36,24 @@ public Server(WorldPersistence worldPersistence, World world, ServerConfig serve
             saveTimer.Elapsed += delegate { Save(); };
         }
 
+        public string SaveSummary
+        {
+            get
+            {
+                // TODO: Extend summary with more useful save file data
+                StringBuilder builder = new StringBuilder();
+                builder.AppendLine($" - Game mode: {world.GameMode}");
+                builder.AppendLine($" - Inventory items: {world.InventoryManager.GetAllInventoryItems().Count}");
+                builder.AppendLine($" - Storage slot items: {world.InventoryManager.GetAllStorageSlotItems().Count}");
+                builder.AppendLine($" - Known tech: {world.GameData.PDAState.KnownTechTypes.Count}");
+                builder.AppendLine($" - Radio messages stored: {world.GameData.StoryGoals.RadioQueue.Count}");
+                builder.AppendLine($" - Story goals unlocked: {world.GameData.StoryGoals.GoalUnlocks.Count}");
+                builder.AppendLine($" - Story goals completed: {world.GameData.StoryGoals.CompletedGoals.Count}");
+                builder.AppendLine($" - Encyclopedia entries: {world.GameData.PDAState.EncyclopediaEntries.Count}");
+                return builder.ToString();
```

Comment:
Would it be possible to have the total that it could be out of?

Reply:
I'll leave this for a new PR since this can be fleshed out a ton from what it is now.